relationships/synthesis/INFRA-KUBERNETES_CONTAINER-to-EXT-SERVICE.yml

```diff
@@ -0,0 +1,33 @@
+relationships:
+  - name: k8sContainerEXTService
```

Comment:
I am not seeing `EXTSERVICE` being the target entity in the definition below

Reply:
entity.type is added in the target.

Reply:
This may now work this way, see internal slack conversation.